code/clue.js

```diff
@@ -5,36 +5,159 @@ const mrGreen = {
   lastName: "Green",
   color: "green",
   description: "He has a lot of connections",
-  age: 45,
+  age: 50,
   image: "assets/green.png",
-  occupation: "Entrepreneur"
+  occupation: "Entrepreneur",
+}
+
+const profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "He is very smart",
+  age: 40,
+  image: "assets/plum.png",
+  occupation: "Professor",
+}
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She is very hard to read",
+  age: 35,
+  image: "assets/scarlet.png",
+  occupation: "Actress",
+}
+
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "Her voice can break glass",
+  age: 45,
+  image: "assets/peacock.png",
+  occupation: "Opera singer",
+}
+
+const mrMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He has a bad temper",
+  age: 70,
+  image: "assets/mustard.png",
+  occupation: "Colonel",
+}
+
+const mrsWhite = {
+  firstName: "Mrs",
+  lastName: "White",
+  color: "white",
+  description: "She knows everything",
+  age: 60,
+  image: "assets/white.png",
+  occupation: "House Maid",
 }
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: "rope",
-  weight: 10
+  name: "Rope",
+  weight: 5,
+}
+
+const knife = {
+  name: "Knife",
+  weight: 3,
+}
+
+const candlestick = {
+  name: "Candlestick",
+  weight: 5,
+}
+
+const dumbbell = {
+  name: "Dumbbell",
+  weight: 10,
+}
+
+const poison = {
+  name: "Poison",
+  weight: 1,
+}
+
+const axe = {
+  name: "Axe",
+  weight: 8,
+}
+
+const bat = {
+  name: "Bat",
+  weight: 7,
+}
+
+const trophy = {
+  name: "Trophy",
+  weight: 5,
+}
+
+const pistol = {
+  name: "Pistol",
+  weight: 2,
 }
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS.
 
+//These are objects, not strings therefore without " "
+
 const suspects = [
   mrGreen,
-  mrsWhite,
   profPlum,
   missScarlet,
   mrsPeacock,
-  mrMustard
-]
+  mrMustard,
+  mrsWhite,
+];
 
-const weapons = []
+//These are objects, not strings therfore without ""
 
-const rooms = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
+
+//These are strings not objects, therefore " "
```

Comment:
Is this comment for the `rooms` array? Then they are strings and not objects. So you have wrote the array correctly!

Reply:
I wrote the comment for myself, to understand what I was doing, and luckily they were right then :)